app/javascript/components/visualization/PlotTitle.js

```diff
@@ -19,5 +21,8 @@ export default function PlotTitle({ cluster, annotation, genes, consensus, subsa
   return <h5 className="plot-title">
     <span className="cluster-title">{contentString} </span>
     <span className="detail"> {detailString}</span>
+    { isCorrelatedScatter &&
+      <span> &nbsp; R<sup>2</sup>={pearsonsR}</span>
```

Comment:
There is a question of whether Pearson's correlation is the most appropriate calculation to make. [correlatePairs](https://rdrr.io/bioc/scran/man/correlatePairs.html) from scran uses Spearman's rank correlation. [Seurat](https://github.com/satijalab/seurat/blob/af2925ce71f51d47cce8e45d94b638ca434a4d33/R/visualization.R#L6898) seems to use Pearsons (the default for "cor") - so there's varying usage and precedent for using Pearson's.

This [explaination](https://towardsdatascience.com/clearly-explained-pearson-v-s-spearman-correlation-coefficient-ada2f473b8) of the difference between the two is compelling that Pearsons will only detect linear relationships well while spearman can detect non-linear but monotonically increasing/decreasing relationships. Since biological relationships aren't reliably linear in nature, Spearman's seems like a better correlation metric to use.

Reply:
Starting with Pearson correlation is reasonable, I think.

But as shown in the original feature request for this ([here](https://github.com/broadinstitute/single_cell_portal/issues/38#issuecomment-436822747)), we may well want _both_ Pearson's _and_ Spearman's (and p-value / adjusted p-value).  This strengthens the case for using a Ruby statistics library instead of us writing the algorithms, but maybe using such a library can weight until we add those other common statistics for gene correlation plots.

What do you think, @jlchang?

Reply:
I've got a meeting with Sarah to talk about other visualizations, I can also ask her

Reply:
Good stuff.  For clarity, let's distinguish the two issues discussed in this thread:

1.  Assertion that writing Pearson correlation coefficient as "R^2" is incorrect, and should be "r".

2. Question about the most appropriate type of correlation for this scenario.